lib/ansible/modules/cloud/ovirt/ovirt_hosts.py

```diff
@@ -40,9 +40,10 @@
     state:
```

Comment:
You miss documentation for `iscsi` attribute. Please add it and please add there a note, that it's used only when state is `iscsidiscover` or `iscsilogin`. and don't forget to add `version_added: "2.4"`